clients/admin-ui/src/features/data-discovery-and-detection/action-center/fields/MonitorFields.const.ts

```diff
@@ -18,29 +20,26 @@ export const RESOURCE_STATUS = [
 ] as const;
 
 export type ResourceStatusLabel = (typeof RESOURCE_STATUS)[number];
-export type ResourceStatusLabelColor =
-  | "nectar"
-  | "red"
-  | "orange"
-  | "blue"
-  | "green";
```

Comment:
This was the source of the problem. We needed a way to enforce brand colors on the Tags.